modules/nested-accordion/widgets/nested-accordion.php

```diff
@@ -151,6 +151,23 @@ protected function register_controls() {
 			]
 		);
 		$this->end_controls_section();
+
+		$this->start_controls_section( 'section_interactions', [
+			'label' => esc_html__( 'Interactions section', 'elementor' ),
+		] );
+
+		$this->add_control( 'default_state', [
+			'label'              => esc_html__( 'Default State', 'elementor' ),
+			'type'               => Controls_Manager::SELECT,
+			'options'            => [
+				'first_expended' => esc_html__( 'First expended', 'elementor' ),
```

Comment:
I think that there is a mistake and it should be `expanded` and not `expended`.
`expend` is like wasting money.
`expand` is to make something bigger.